oraibtcbridge/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "oraibtcbridge",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://orai.io/",
+  "pretty_name": "Orai Bitcoin Bridge",
+  "chain_id": "oraibtc-subnet-1",
+  "bech32_prefix": "oraibtc",
+  "daemon_name": "nomic",
```

Comment:
Pretty sure this daemon name (and probably the daemon name in the other file are left over value and are incorrect). I'm guessing you copied Gravity Bridges' and Nomic's chain.json files and updated value here and there.